daemon/logger/jsonfilelog/jsonlog/jsonlogbytes_test.go

```diff
@@ -29,6 +29,9 @@ func TestJSONLogsMarshalJSONBuf(t *testing.T) {
 		{Log: []byte{0x7F}}:            `^{\"log\":\"\x7f\",\"time\":`,
 		// with raw attributes
 		{Log: []byte("A log line"), RawAttrs: []byte(`{"hello":"world","value":1234}`)}: `^{\"log\":\"A log line\",\"attrs\":{\"hello\":\"world\",\"value\":1234},\"time\":`,
+		// with Tag set
+		{Log: []byte("A log line with tag"), RawAttrs: []byte(`{"hello":"world","value":1234}`)}: `^{\"log\":\"A log line with tag\",\"attrs\":{\"hello\":\"world\",\"value\":1234},\"time\":`,
+		//{Log: []byte("A log line with tag"), RawAttrs: []byte(`{"tag":"test-tag"}`)}: `^{\"log\":\"A log line with tag\",\"attrs\":{\"tag\":\"test-tag\"}\",\"time\":`,
```

Comment:
Looks like this was left behind for debugging?

Reply:
@thaJeztah Updated 😅 Thanks!